### DISCUSSION TITLE
Post Build Issues

### DISCUSSION THREAD

**Discussion by BenAigan** [git-for-windows/git#3479] [Q&A]
I previously built Git for Windows V2.28.0 but it says git-parse-remote.exe not found when trying to use submodules so trying to build newer versions. 

I am (not a developer) building Git for Windows inside my firm, basically I clone code, open git-bash and run "sdk build" which runs successfully.

However when I try to make a portable release it states multiple files missing:

e.g. sdk cd build-extra
./portable-release.sh releaseVersion

mingw64/libexec/git-core/git-add.exe : no more files
(and many more) 

What am I missing? Any help greatly appreciated.

**Comment by dscho** [comment]
>  ./portable-release.sh releaseVersion

You mean `./portable/release.sh <version>`, right? Try calling `sh -x ./portable/release.sh 0-test`, that hopefully provides hints what is going wrong.
